app/views/subject_plans/semester_card/_add_subject_form.html.erb

```diff
@@ -17,7 +17,7 @@
     { data: { subject_selector_target: "select", action: "subject-selector#onChange" }, class: "invisible absolute" }
   ) %>
   <%= form.hidden_field :semester, value: semester %>
-  <%= form.button type: :submit, class: "cursor-pointer py-2 ps-3 material-icons text-gray-400 disabled:opacity-50 disabled:cursor-not-allowed", disabled: true, data: { subject_selector_target: "submitButton" } do %>
+  <%= form.button type: :submit, class: "cursor-pointer py-2 ps-3 material-icons text-gray-400 disabled:opacity-50 disabled:cursor-not-allowed", disabled: true, data: { subject_selector_target: "submitButton", action: "planner-loading#start" } do %>
```

Comment:
otra opción capaz puede ser poner `data-action="turbo:submit-start->planner-loading#start"` en el `div#planner`

creo que no serviría para el drag and drop, pero capaz se puede hacer algo para que lo incluya.
Y también incluiría otros request con turbo adentro del div, no sé si tenemos algún otro